tajo-client/src/main/java/org/apache/tajo/client/CatalogAdminClientImpl.java

```diff
@@ -164,21 +166,22 @@ public TableDesc createExternalTable(final String tableName, final Schema schema
       throw new RuntimeException(e);
     }
 
-    if (isThisError(res.getState(), Errors.ResultCode.DUPLICATE_TABLE)) {
-      throw new DuplicateTableException(res.getState());
-    }
+    throwsIfThisError(res.getState(), DuplicateTableException.class);
+    throwsIfThisError(res.getState(), InsufficientPrivilegeException.class);
```

Comment:
The above two exceptions should be properly thrown when TajoMasterClientService.createExternalTable() is called.
For more detailed, these exceptions should be caught at the following code (TajoMasterClientService::887), and then thrown properly to handle them here.
```
TableDesc desc = context.getGlobalEngine().getDDLExecutor().getCreateTableExecutor().create(
            queryContext,
            request.getName(),
            null,
            schema,
            meta,
            path.toUri(),
            true,
            partitionDesc,
            false
        );
```